app/templates/course/stage/code-examples.hbs

```diff
@@ -1,4 +1,11 @@
-<div class="pt-8 pb-32" {{did-update this.loadSolutions this.courseStage}} {{did-insert this.loadSolutions}} data-test-code-examples-tab>
+<div
+  class="pt-8 pb-32"
+  {{did-update (fn (mut this.stageIncompleteModalWasDismissed) false) this.courseStage}}
```

Comment:
I might be wrong about this, but even if a property on `courseStage` changes, this would fire. (`courseStage.instructionsMarkdown`, for example, or `courseStage.solutionCounts`)

Can we confirm this? In the past, I think I've always ended up storing `courseStage` as a property in the component/controller and then on each callback I'd compare the received courseStage against the stored property and not do anything unless it has changed

Reply:
added another test to check for this